view/index.html

```diff
@@ -59,13 +59,13 @@ <h3 class="panel-title">Ayuda</h3>
                         </li>
                      </ul>
                   </li>
-                  {if="isset($_GET['page'])"}
+                  {if="filter_input(INPUT_GET, (string)'page')"}
                   <li>
                      <b>¿Eres programador y estas intentando hacer un plugin?</b>
                      <ul>
                         <li>
                            Entonces te comento que lo que ha pasado es que no encuentro
-                           el archivo <b>{$_GET['page']}.php</b>, que debería estar en
+                           el archivo <b>{filter_input(INPUT_GET, (string)'page')}.php</b>, que debería estar en
```

Comment:
Got it and Do it! He corregido aquí también que al final los casting no son necesarios corrigiendo la documentación y la declaración de los parámetros de la función.